lib/models/navigation/page_data/page_data.dart

```diff
@@ -0,0 +1,13 @@
+library page_data;
```

Comment:
This is the parent class of all the page data's.  Polymorphism in built_value is a bit weird, there may be a better way

Reply:
cool got it